client/src/group/Group.ts

```diff
@@ -34,7 +37,11 @@ export class Group implements GroupInterface {
     }
     // todo: fix type
     const overview: any = await this.newsieClient.over(`${group.low}-${group.high}`);
-    const articlesByNumber: Article[] = overview.articles
+    const groupCache = await GroupCache.instance();
+    await groupCache.persistOverArticles(this.host, this.name, overview.articles);
+    const articles = await groupCache.retrieveOverArticles(this.host, this.name);
```

Comment:
Nice job on this commit, I can confirm that I can access the article contents now offline. What would be cool (maybe I'll do that), if we could cache the group overview too, thus making our app offline accessible. 

@PeterTheOne I think on this line here the cache retrieval should be before communicating with newsie, like we do for the article body, right?

Reply:
Complete offline access would be awesome, feel free to work on that. I'm happy to help.

@Elektropepi Here we need to check if there are new articles that we don't know of yet, there might be new ones. For body we already know that there should be an article with that id. It could be a lot smarter and just fetch from the articleNumber that we already have. And for total offline access fetch new articles should be able to fail and still work.

For now I think we can merge it and improve later, or what do you think?

Reply:
Sure